index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="/css/style.css">
+    
+</head>
+<body>
+    <main class="main">
+        <img src="/assets/images/bar.png" alt="image">
+    </main>
+<div class="section">
+    <div class="width"></div>
+    <div class="Home-content">
+        <h1>Hey There.<br>I'm Nonhlanhla Mndebele</h1>
+        <h2>I'am a Software Developer</h2>
+        <h3>I can help you build a product, feature or website look<br>
+            through some of my work and experience! If you like<br>
+            what you see and have a project you need coded,<br>
+            don't hesitate to contact me.</h3>  
+        <ul class="social">
+            <li class="github a"><a href="ct"><img src="/assets/icons/git.png" alt="img"></a></li>
+            <li class="linkedin a"><a href="ct"><img src="assets/icons/i.png" alt="img"><span><img src="assets/icons/n.png" alt="img"></span></a></li>
+            <li class="angellist a"><a href="ct"><img src="assets/icons/two.png" alt="img"></a></li>
+            <li class="twitter a"><a href="ct"><img src="assets/icons/twet.png" alt="img"></a></li>
+            <li class="medium a"><a href="ct"><img src="assets/icons/med.png" alt="img"></a></li>
+        </ul>
+
+    </div>
+</div> 
```

Comment:
- [ ] Kindly be noticed that you should wrap this part inside a `section` tag.
- [ ] Also note that instead of using `h3` in the part of `I can help you build a product` use `p` tag.
- [ ] Also please follow the design of the Figma in your design.

Your Design | Figma Design
---------------- | ------------------
![Screenshot from 2022-05-26 23-01-22](https://user-images.githubusercontent.com/33752735/170579764-843be7ed-db0d-4aa3-afec-4910d8389ded.png) | ![Screenshot from 2022-05-26 23-02-35](https://user-images.githubusercontent.com/33752735/170579786-dfb2dcf0-c328-48b9-b098-b1443cccc23b.png)

- [ ] Please ignore the `N`, `A`, and `O` icons in the figma design.